### DISCUSSION TITLE
Automatic convert to kepub? When?

### DISCUSSION THREAD

**Discussion by thehijacker** [crocodilestick/Calibre-Web-Automated#75] [Q&A] [answered]
I had Calibre-Web and used Calibre in Windows desktop for uploads. I always manually converted epub to kepub for Kobo uploading. Now using only Calibre-Web and as soon as I inputed the path to kepubify, it automatically created kepub files for books that did not have it yet.

When I ingest new book via ingest folder, it doesn't created kepub file automatically. So what is the rule?

**Comment by jmarmstrong1207** [comment, marked as answer]
When kepubify path is set in the calibre-web settings, it will automatically convert to kepub when syncing to Kobo. calibre-web-automated's script itself will only convert all files to epub, which will then be converted to kepub when syncing to Kobo
